.github/changelog.md

```diff
@@ -4,10 +4,44 @@ Release History
 [Release 3.10.1](https://github.com/GUDHI/gudhi-devel/releases/tag/tags%2Fgudhi-release-3.10.1)
 -----------
 
-Release date: June 2024
+Release date: February 2025
 
 Below is a list of changes made since GUDHI 3.10.1:
 
+- [Delaunay complex](https://gudhi.inria.fr/python/3.11.0rc1/delaunay_complex_user.html)
+     - The Delaunay complex can be equipped with different filtrations:
+          * Delaunay complex (no filtration values computed)
+          * Delaunay-Čech complex (using minimal enclosing ball)
+          * Alpha complex (moved in this new section)
+     - The Delaunay-Čech and Alpha complex can output square, or not square, filtration values
+     - An incremental version of the Delaunay complex (only in C++)
+
+- [Rips complex persistence scikit-learn like interface](https://gudhi.inria.fr/python/3.11.0rc1/rips_complex_sklearn_itf_ref.html)
+     - A binding to [Ripser](https://github.com/Ripser/ripser) when it accelerates the computation
+
+- [Persistence graphical tools](https://gudhi.inria.fr/python/3.11.0rc1/persistence_graphical_tools_user.html)
+     - Can now handle scikit-learn like interfaces outputs as inputs
+
+- [Simplex tree](https://gudhi.inria.fr/doc/3.11.0rc1/class_gudhi_1_1_simplex__tree.html)
+     - Can now store additionnal data on each simplices (only in C++)
```

Comment:
each simpl**ex** (simplices is the plural)

Reply:
Fixed on #1178